src/website/app/demos/Table/examples/sortableControlled.js

```diff
@@ -28,65 +61,72 @@ be managed by the application as a controlled component via the control prop,
           this.state = initialState;
 
           this.handleSort = this.handleSort.bind(this);
-          this.sortFruitsAscending = this.sortFruitsAscending.bind(this);
-          this.sortFruitsDescending = this.sortFruitsDescending.bind(this);
           this.reset = this.reset.bind(this);
+          this.sortDairyAscending = this.sortDairyAscending.bind(this);
+          this.sortDairyDescending = this.sortDairyDescending.bind(this);
+
+          this.sort = memoizeOne(sortFn)
```

Comment:
Since data is an object, I think you'l need to provide your own equality function as a 2nd parameter to `memoizeOne` and use `deepEqual` with that.